appcues/src/main/java/com/appcues/di/Bootstrap.kt

```diff
@@ -26,22 +27,22 @@ internal object Bootstrap {
     )
 
     fun createScope(context: Context, config: AppcuesConfig): AppcuesScope {
-        return start(ContextWrapper(context), modules) {
+        return start(modules) {
             scoped { config }
+            scoped { ContextWrapper(context) }
```

Comment:
moved context related dependencies from Start to this block to allow for customization during test